core/drivers/ls_sfp.c

```diff
@@ -0,0 +1,423 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2022 Microsoft
+ *
+ * Driver for the NXP LX2160A-series Security Fuse Processor (SFP).
+ */
+
+#include <assert.h>
+#include <drivers/ls_sfp.h>
+#include <io.h>
+#include <kernel/boot.h>
+#include <kernel/dt.h>
+#include <kernel/tee_time.h>
+#include <libfdt.h>
+#include <mm/core_memprot.h>
+#include <util.h>
+
+/**
```

Comment:
`/* `in place of `/**` for all comments section.
Please correct in each function information

Reply:
I've been following the [linux style guide](https://www.kernel.org/doc/html/latest/doc-guide/kernel-doc.html) when it comes to commenting syntax. Is there an optee rule exception for commenting I'm not aware of? The [optee coding style page](https://optee.readthedocs.io/en/latest/general/coding_standards.html) doesn't mention anything about it.

From the kernel guide: 

> "Every function that is exported to loadable modules using EXPORT_SYMBOL or EXPORT_SYMBOL_GPL should have a kernel-doc comment."
> "The opening comment mark /** is used for kernel-doc comments."

Reply:
I don't think we follow kernel style comments here in OP-TEE.
Yes, It is not mentioned in the OP-TEE coding style, but I don't see comments starting with `/**` anywhere in the OP-TEE code.
Maybe @jforissier is the best person to confirm.

Reply:
`/*` and `/**` are used inconsistently in the OP-TEE source code. My recommendation is to stick with the Linux convention and therefore use `/**` even though such blocks are technically not used to produce any external documentation at the moment.